<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContactsWindow</class>
 <widget class="QDialog" name="ContactsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Contacts</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../usr/local/share/ksms/icon.png</normaloff>../../../../usr/local/share/ksms/icon.png</iconset>
  </property>
  <widget class="QTableView" name="ContactsList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>451</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="autoScroll">
    <bool>true</bool>
   </property>
   <property name="tabKeyNavigation">
    <bool>false</bool>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="cornerButtonEnabled">
    <bool>false</bool>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QPushButton" name="addContactButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>381</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Contact</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okayButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>470</y>
     <width>81</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Okay</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameTextEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="numberTextEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>470</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Number</string>
   </property>
  </widget>
  <zorder>ContactsList</zorder>
  <zorder>okayButton</zorder>
  <zorder>nameTextEdit</zorder>
  <zorder>numberTextEdit</zorder>
  <zorder>addContactButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
